program/tests/tests/collect_validator_fee.rs

```diff
@@ -86,7 +86,7 @@ async fn test_collect_validator_fee() {
     // configuration yields, so we have to deal with it.
     context.update_exchange_rate().await;
     let rewards = context.collect_validator_fee(validator.vote_account).await;
-    assert_eq!(rewards, Lamports(1246_030_107_210));
+    assert_eq!(rewards, Lamports(1_246_030_615_387));
```

Comment:
Is this because of https://github.com/solana-labs/solana/issues/18942?

Reply:
Might be, I’m not sure ... if you would like to know, you can bisect Solana :see_no_evil: